src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int supply = 0;
+    private int buy = 0;
+    private int result = 0;
+    private final String split = ",";
+    private int index = 1;
```

Comment:
These fields are method implementation details. Seems like it won't work properly if you call the getStatistic method several times. Move them to the method level